public/sass/pages/_explore.scss

```diff
@@ -55,6 +55,25 @@
     margin-top: 2 * $panel-margin;
   }
 
+  .time-series-disclaimer {
+    width: 300px;
```

Comment:
I'd make that `min-width`, and then give it an `em` value for scaling.

Reply:
em value? We don't use em values anywhere. We used to use mostly rems but are moving from rems to pixels as the argument to use rems are mostly gone now that browser handle the scaling correctly for both rems & pixels and defining size in pixels is so much easier and accurate (create less floating point pixels and other misalignments that rems can cause)